src/main/java/com/minecolonies/commands/MinecoloniesCommand.java

```diff
@@ -17,6 +17,8 @@
     private final ImmutableMap<String, ISubCommand> subCommands =
       new ImmutableMap.Builder<String, ISubCommand>()
         .put("colonies", new ColoniesCommand(DESC))
```

Comment:
But why can't we replace `.put("colonies",....` with `.put(ColoniesCommand.DESC,....`?